.golangci.yml

```diff
@@ -271,6 +271,8 @@ linters:
       - linters:
           - staticcheck
         text: 'SA1019: .*(res|result|i|j)\.Requeue is deprecated: Use `RequeueAfter` instead'
+        # TODO: Remove Requeue step by step
+        path: internal/controllers/(extensionconfig|machinepool)|util/util.go
```

Comment:
Let's change util/util.go to a local nolint in LowestNonZeroResult in the next PR (it's fine to combine it with a change to another controller)

We have to keep handling Requeue there until the field is removed by controller-runtime

(and I don't want to keep the exclude here so long because I'm not sure how long it will take to drop the field in CR)